public/visController.js

```diff
@@ -314,17 +316,19 @@ define(function (require) {
       if (!poiLayerArray) return;
 
       poiLayerArray.forEach(layerParams => {
-        layerParams.enabled = uiState.get(layerParams.id);
+        layerParams.enabled = sirenSessionState.get(layerParams.id);
```

Comment:
I think it would be enough to
```suggestion
        layerParams.enabled = sirenSessionState.get(layerParams.id) || true;
```
without setting it to true if undefined.

Reply:
if it's undefined it's not enabled meaning that it won't be drawn. I think this is what we want

Reply:
Hmm, you're making me think for sure :). 

I think if the sirenSessionState.get() returns false, then the true will be used as per picture? If so, we don't want that as it would mean that even layers that are not enabled according to the uiState would be enabled. 
The undefined is used for the case of a new layer. Where it should be visible on first load and set to true in uiState, again just on first load, if a user toggles layer off in layer control, the it will be set to false in the uiState. 

![image](https://user-images.githubusercontent.com/36197976/87033230-bcf00080-c1dd-11ea-904b-acf736a5f05f.png)

Reply:
OPS! maybe:
```javascript
const layerEnabled = sirenSessionState.get(layerParams.id);
layerParams.enabled = layerEnabled === undefined ? true : layerEnabled;
```

Or just leave it as it is :D